runtime/src/migrations.rs

```diff
@@ -30,6 +30,34 @@
 
 use crate::*;
 
+#[cfg(feature = "wip")] // dex-kusd
+pub type Migrations = (
+    assets::migration::register_asset::RegisterAsset<
+        Runtime,
+        KGOLDAssetId,
+        KGOLDAssetName,
+        KGOLDAssetSymbol,
+        PredefinedAssetOwnerAccountId,
+    >,
+    assets::migration::register_asset::RegisterAsset<
+        Runtime,
+        KXORAssetId,
+        KXORAssetName,
+        KXORAssetSymbol,
+        PredefinedAssetOwnerAccountId,
+    >,
+    assets::migration::register_asset::RegisterAsset<
+        Runtime,
+        KARMAAssetId,
+        KARMAAssetName,
+        KARMAAssetSymbol,
+        PredefinedAssetOwnerAccountId,
+    >,
+    kensetsu::migrations::v1_to_v2::UpgradeToV2<Runtime>,
```

Comment:
Upgrade with Kensetsu migration v1 to v2 and registration of tokens `KGOLD`, `KXOR`, `KARMA` will be applied in release 3.5.0 Tokenomics Upgrade. You can safely delete them in this `wip`.

```suggestion
```

And when 3.5.0 finally released, you can delete these migrations at all.